src/main/java/com/bookstore/repository/BookRepository.java

```diff
@@ -1,7 +1,11 @@
 package com.bookstore.repository;
 
 import com.bookstore.model.Book;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
 import org.springframework.data.jpa.repository.JpaRepository;
 
 public interface BookRepository extends JpaRepository<Book, Long> {
+
+    Page<Book> findAll(Pageable pageable);
```

Comment:
I think JpaRepository has already had this method, and you don't need this bu your hand to your repository. Please double check this